python/requirements.txt

```diff
@@ -58,6 +58,6 @@ starlette
 typer
 fsspec
 pandas>=1.3
-pydantic>=2.5.0,<2
+pydantic!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*
```

Comment:
Thanks for the suggestion! I added the upper bound. I'll omit the lower bound for now since we haven't done rigorous testing to decide on a good lower bound.